frontend/src/common/top_bar.tsx

```diff
@@ -38,7 +38,12 @@ export function MeMenu() {
           Change Password
         </SecondaryButton>
         <PrimaryButton
-          onClick={() => {
+          onClick={async () => {
+            try {
+              await logout({});
+            } catch (e) {
+              /* empty */
+            }
             storeAuthToken(undefined);
             mutate();
```

Comment:
I think this call is redundant. It tries to get info about the user and gets 404 that user is not found (because the token was erased even with `storeAuthToken(undefined)`)

Reply:
Yes, it is probably reduntant in the current setup, especially since we call `window.location.reload();` a few lines later: `mutate` marks the swr cache as expired and refetches the data[^mutate], but we currently do not keep the data in the browser cache across reloads. I would still keep it, as it seems to me to be cleaner to mark the data as invalid. Especially if we ever implement some form of data sharing across tabs/reloads, this might be needed.

[^mutate]: https://swr.vercel.app/docs/mutation#revalidation